aspnetcore/signalr/configuration.md

```diff
@@ -0,0 +1,202 @@
+---
+title: ASP.NET Core SignalR configuration
+author: rachelappel
+description: Configure ASP.NET Core SignalR Apps
+manager: wpickett
+monikerRange: '>= aspnetcore-2.1'
+ms.author: rachelap
+ms.date: 06/30/2018
+ms.prod: asp.net-core
+ms.technology: aspnet
+ms.topic: article
+uid: signalr/configuration
+---
+
+# ASP.NET Core SignalR configuration
+
+## JSON/MessagePack Serialization Options
+
+ASP.NET Core SignalR supports two protocols for encoding messages, JSON and MessagePack. Each protocol has configuration options that can be used to configure serialization.
+JSON Serialization Options
```

Comment:
"###"?

Reply:
3 #'s are a H3 subheading (these under messagepack). Move to H2 (##)?